ts/src/time-profiler.ts

```diff
@@ -169,3 +170,4 @@ export function v8ProfilerStuckEventLoopDetected() {
 
 export const constants = {kSampleCount};
 export {LabelSet};
+export {getNativeThreadId};
```

Comment:
This looks weird, wouldn't this be equivalent?
```suggestion
export {getNativeThreadId, LabelSet};
```